pkg/virt-controller/services/rendercontainer_test.go

```diff
@@ -92,11 +92,6 @@ var _ = Describe("Container spec renderer", func() {
 				specRenderer = NewContainerSpecRenderer(containerName, img, pullPolicy, WithCapabilities(simplestVMI()))
 			})
 
-			It("must request to drop the NET_RAW capability", func() {
-				Expect(specRenderer.Render(exampleCommand).SecurityContext.Capabilities.Drop).Should(
-					Equal([]k8sv1.Capability{CAP_NET_RAW}))
```

Comment:
@xpivarc I'm not sure how this is related to runtimes, since it's only a unit test.
Bringing the test back in with an `Or()` probably makes sense, just to avoid future regressions.

Reply:
Not the unit test but the matter it is testing. Runtimes can have default capabilities and I suspect that is why we explicitly were asking to drop this capability(not sure).  The `Or` makes the most sense to be as long as we don't get input from the networking team.

Reply:
When running this test, `Capabilities.Drop` is empty. Is that a bug?

Reply:
Ah! This is actually tested right below. This `specRenderer` was just not created with `WithDropALLCapabilities()`.
Removing this test actually makes sense then IMO.